### PR TITLE
api changes in 6.14

### PR DESCRIPTION
**Pull request by pondrejk**
Updated the path list with the 6.14 changes, awaiting approval on sat-eng list for one remaining bit

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/test_api_endtoend.py -k get_links

**Review by jyejare — CHANGES_REQUESTED** [review]
Test in PRT failed with assertion error for api path mismatch!

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/test_api_endtoend.py -k get_links

File: tests/foreman/endtoend/test_api_endtoend.py [hunk 1]
```diff
@@ -432,6 +432,8 @@
         '/api/hostgroups/multiple_play_roles',
         '/api/hostgroups/:id/ansible_roles',
         '/api/hostgroups/:id/assign_ansible_roles',
+        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
+        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
```
Comment (Gauravtalreja1) [inline tests/foreman/endtoend/test_api_endtoend.py:436]: Non-blocking, this is just for readability, same can be done for hosts entity
```suggestion
        '/api/hostgroups/:id/ansible_roles',
        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
        '/api/hostgroups/:id/assign_ansible_roles',
```

**Comment by pondrejk** [comment]
prt build 2781 now passing:
```

pytest tests/foreman/endtoend/test_api_endtoend.py -k get_links --external-logging --junit-xml=sat-results.xml -o junit_suite_name=sat-result
 ============================= test session starts ==============================
 platform linux -- Python 3.11.3, pytest-7.3.2, pluggy-1.2.0
 Mandatory Requirements are up to date.
 Optional Requirements Mismatch: sphinx-autoapi==2.1.1 pre-commit==3.3.3 sphinx==7.0.1 manage==0.1.15 redis==4.5.5 flake8==6.0.0 pytest-cov==3.0.0
 To update mismatched requirements, run the pytest command with '--upgrade-required-reqs' OR '--upgrade-all-reqs' option.
 shared_function enabled - OFF - scope:  - storage: file
 rootdir: /opt/app-root/src/robottelo
 configfile: pyproject.toml
 plugins: ibutsu-2.2.4, mock-3.11.1, reportportal-5.1.9, services-2.2.1, xdist-3.3.1
 collected 6 items / 5 deselected / 1 selected
 
 tests/foreman/endtoend/test_api_endtoend.py .                            [100%]
```

File: tests/foreman/endtoend/test_api_endtoend.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/foreman/endtoend/test_api_endtoend.py]: @pondrejk Thanks for addressing this, as you commited a suggestion which doesn't updates host entity, can you update it manually please?

Comment (pondrejk) [reply tests/foreman/endtoend/test_api_endtoend.py]: ah I see, updated, thanks!

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**